app/Theme/AbstractTheme.php

```diff
@@ -945,24 +945,23 @@ protected function individualBoxMenu(Individual $individual) {
 	 * @return Menu[]
 	 */
 	protected function individualBoxMenuCharts(Individual $individual) {
-		$menus = array_filter(array(
-			$this->menuChartAncestors($individual),
-			$this->menuChartCompact($individual),
-			$this->menuChartDescendants($individual),
-			$this->menuChartFanChart($individual),
-			$this->menuChartHourglass($individual),
-			$this->menuChartInteractiveTree($individual),
-			$this->menuChartPedigree($individual),
-			$this->menuChartPedigreeMap($individual),
-			$this->menuChartRelationship($individual),
-			$this->menuChartTimeline($individual),
-		));
-
-		usort($menus, function (Menu $x, Menu $y) {
-			return I18N::strcasecmp($x->getLabel(), $y->getLabel());
-		});
+		$menus = array();
+		foreach (Module::getActiveCharts($this->tree) as $chart) {
+			$menu = $chart->getChartMenu($individual);
```

Comment:
Some charts are specific to an individual.  e.g. pedigree.
Others are not.  e.g. statistics.

The non-individual charts do not belong here.